mlir/test/Dialect/Tosa/constant-clamp-opt.mlir

```diff
@@ -0,0 +1,66 @@
+// RUN: mlir-opt --split-input-file -verify-diagnostics --tosa-layerwise-constant-fold %s | FileCheck %s
+
```

Comment:
Could you add a test case where the clamp min/max are not representable by the output type? I saw that the implementation is very careful, and it would be great to ensure that this stays working in the future.

Reply:
Added one for float/int in 18c4a2ef58774db70c007ec6a516548bb988b7de.